src/table/utils/copy-utils.ts

```diff
@@ -0,0 +1,14 @@
+const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
+  const target = evt.target as HTMLElement;
+  const value =
+    isHeadCell && target.children[0].firstChild
+      ? target.children[0].firstChild.textContent
+      : target.firstChild && target.firstChild.textContent;
```

Comment:
you could use optional chaining here

Reply:
`target.firstChild && target.firstChild.textContent` will always return the same thing as `target.firstChild?.textContent`, it is just a shorter way of writing it